www-servers/nginx-unit/nginx-unit-1.14.0.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=(python2_7 python3_{5,6,7})
```

Comment:
From the changelog, v1.13.0 introduced Py3.8 support. Can you please test the package with that and add it accordingly?

Reply:
I'll test with Python 3.8.

Reply:
Python 3.8 looking good, and I updated the ebuild accordingly.